src/TodoListItem.js

```diff
@@ -1,9 +1,16 @@
 import React from "react";
+import styles from "./TodoListItem.module.css";
 
 export default function TodoListItem({ todo, removeTodo }) {
   return (
-    <li>
-      {todo.title} <button onClick={() => removeTodo(todo.id)}>Remove</button>
+    <li className={styles.ListItem}>
+      <button
+        className={styles.RemoveButton}
+        onClick={() => removeTodo(todo.id)}
+      >
+        Remove
+      </button>{" "}
```

Comment:
One of the things css can help you do is not need to rely on things like this empty space string between the elements: 

```suggestion
      </button>
```

Reply:
We can handle this inside the `.listItem` class, potentially by adding margins after the button, or via a flexbox way I'll show you